content/en/real_user_monitoring/guide/upload-javascript-source-maps.md

```diff
@@ -15,7 +15,7 @@ further_reading:
 If your front-end Javascript source code is minified, you will need to upload your source maps to Datadog so that we are able to deobfuscate your different stack traces. For a given error, you will then get access to the file path, the line number, as well as a code snippet for each frame of the related stack trace.
 
 ## Instrument your code
-You must configure your Javascript bundler so that, when minifying your source code, it generates source maps which directly include the related source code in the `sourcesContent` attribute. Check below some configurations for popular Javascript bundlers.
+You must configure your Javascript bundler so that, when minifying your source code, it generates source maps which directly include the related source code in the `sourcesContent` attribute. In addition, you should make sure that the size of each source map augmented with the size of the related minified file does not exceed __our limit of 50mb__. Check below some configurations for popular Javascript bundlers.
```

Comment:
```suggestion
You must configure your Javascript bundler so that, when minifying your source code, it generates source maps which directly include the related source code in the `sourcesContent` attribute. In addition, ensure that the size of each source map augmented with the size of the related minified file does not exceed __our limit of 50mb__. Check below for some configurations for popular Javascript bundlers.
```